recipes/torch-spline-conv/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "torch_spline_conv" %}
+{% set version = "1.2.0" %}
+
+{% set pytorch_spline_conv_proc_type = "cpu" if cuda_compiler_version == "None" else "gpu" %}  # [linux or win]
+{% set pytorch_spline_conv_proc_type = "cpu" %}                                                # [osx]
+
+package:
+  name: py{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b7a1788004f6c6143d47040f2dd7d8a579a0c69a0cb0b5d7537416bf37c082a5
+
+build:
+  number: 0
+  skip: True  # [win]
+
+outputs:
+  # A meta-package to select CPU or GPU pytorch_spline_conv build.
+  - name: pytorch_spline_conv_proc
+    version: 1.0.0
+    build:
+      number: 0
+      string: "{{ pytorch_spline_conv_proc_type }}"
+    test:
+      commands:
+        - exit 0
```

Comment:
This needs to be a real test of some kind

Reply:
I followed the recipe format for [nvidia-apex](https://github.com/conda-forge/nvidia-apex-feedstock/blob/master/recipe/meta.yaml) which uses this test command because it's just a meta-package to select CPU or GPU.

Reply:
Ahh OK